.travis.yml

```diff
@@ -1,18 +1,17 @@
 language: python
 python:
-- 3.5.0
-- 3.5.2
-- 3.5-dev
+- 3.5
 - 3.6
-- 3.6-dev
-- 3.7-dev
+- 3.7
+- 3.8-dev
 sudo: false
-dist: trusty
+dist: xenial
 matrix:
   include:
-  - os: linux
-    language: generic
-    env: USE_PYPY_RELEASE_VERSION=5.9-beta
+  # Pypy seems to have a problem with getfullargspec
+  # - os: linux
+  #   language: generic
+  #   env: USE_PYPY_RELEASE_VERSION=5.9-beta
```

Comment:
5.9-beta is pretty old. Nowadays, you can just use `pypy3.5` in the list of Travis python versions to get the latest pypy without all the fuss.

If the latest pypy also has this problem, it would be nice to file a bug with them.